slides/webpack.config.js

```diff
@@ -29,9 +29,10 @@ module.exports = {
     new HtmlWebpackPlugin({
       template: './index.html'
     }),
-    new CopyWebpackPlugin([ 
-      { from: { glob: 'content/*.md' } },
-      { from: { glob: 'content/images/*' } }
+    new CopyWebpackPlugin([
+      { from: { glob: 'content/**/*.md' } },
+      { from: { glob: 'content/images/*' } },
+      { from: { glob: 'content/bios/images/*' } }
```

Comment:
Fine by me - I'm partial to keeping assets separate from the content

Maybe it makes sense to have a "bios" subdirectory in the original images folder?